docs/API.md

```diff
@@ -313,6 +313,17 @@ tracer.use('express', { headers: ['x-my-tagged-header'] })
 
 Each integration also has its own list of default tags. These tags get automatically added to the span created by the integration.  Some have additional configuration settings to determine traced behavior.  These are configured by providing an object of the form `{ optionOneName: optionOneValue, optionTwoName: optionsTwoValue }`.
 
+#### adonisjs
+
+##### Tags
+
+| Tag              | Description                                               |
+|------------------|-----------------------------------------------------------|
+| http.url         | The complete URL of the request.                          |
+| http.method      | The HTTP method of the request.                           |
+| http.status_code | The HTTP status code of the response.                     |
+| http.headers.*   | A recorded HTTP header. 
```

Comment:
This warrants a configuration options section

Reply:
OK. Example please. Not sure I have a clear picture in my mind.

Reply:
There are several in this document.